src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck start working!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stop working!");
```

Comment:
Similarly, the stop message should be more informative. You could change it to 'Truck has stopped working.' to clearly indicate the machine type and its current state.